src/problem1.rb

```diff
@@ -0,0 +1,17 @@
+class Problem1
+  def sumOfAllEvenNumbers(values)
+    response = 0
+    if(!values.nil? && (values.instance_of?(Array)))
+      if(values.all? {|i| i.is_a?(Integer)})
+        values.select {|i| i % 2 == 0}.each {|number|
+          response = response + number;
```

Comment:
Check the array documentation and figure out if there is a better way to sum up elements in an array